packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -148,8 +148,8 @@ export class AndroidSdkTools {
     const apkSignerParams = [
       `sign --ks ${keystore}`,
       `--ks-key-alias ${alias}`,
-      `--ks-pass pass:${ksPass}`,
-      `--key-pass pass:${keyPass}`,
+      `--ks-pass 'pass:${ksPass}'`,
```

Comment:
re: @PEConn's comment on escaping single quotes, we'd need something like this. Worth adding some tests:

```suggestion
      `--ks-pass 'pass:${ksPass.replace(/'/g,'\\\'')}'`,
```